### PR TITLE
add post about xsd and xslt

### PR DESCRIPTION
**Pull request by jeroenooms**
Can you have a look @cboettig @sckott ? Thanks :)

### CODE REVIEW

**Comment by sckott** [comment]
okay if we put up on Tuesday?  a post by Dan scheduled to go up tomorrow

File: _posts/2017-01-09-xslt-release.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+- ropensci
+- packages
+- xml
+- xslt
+- R
+---
+
+This week an update for [xml2](https://cran.r-project.org/web/packages/xml2/index.html) and a new [xslt](https://cran.r-project.org/web/packages/xslt/index.html) package have appeared on CRAN. A full announcement for xml2 version 1.1 will appear on the rstudio blog. This post explains xml *validation* (via xsd schema) and xml *transformation* (via xslt stylesheets) which have been added in this release.
+
+XML schemas and stylesheets are not exactly new; both [xslt 1.1](https://www.w3.org/TR/xslt11/) (2001) and [xsd 1.0](https://www.w3.org/TR/xmlschema-1/) (2004) have been available in browsers for over a decade. Revised specifications for xsd/xslt are still developed, but not widely implemented due to declined popularity of xml itself. However they are still needed sometimes now available in R.
```
Comment (sckott) [inline _posts/2017-01-09-xslt-release.md]: > However they are still needed sometimes now available in R.

clarify, not sure what this means

**Comment by jeroenooms** [comment]
OK bumped the date and improved the wording a bit.

**Comment by sckott** [comment]
thanks, will post tmrrw morning about 815 PST (after mailchimp email goes out with todays post)

**Pull request merged**